app/lib/cooldown.js

```diff
@@ -6,24 +6,40 @@ module.exports = Base => class extends Base {
         
         //object containing the cooldown times for the commands. new commands would need a new property added here.
         this.coodownTimes = {
-            'home': 1000 * 60 * 15,
-            'warp': 1000 * 60 * 15
+            '!home': 1000 * 60 * 15,
+            '!warp': 1000 * 60 * 15,
+            '!location': 1000 * 60 * 15
         }
     }
 
-    //Check cooldown. place before code you want to execute.
-    cooldownCheck(command, playerName) {
+    //Check cooldown. place before code you want to execute. optionalDisplayCommandName lets you display a different name for a command to the player, and optionalRawMessage allows you to set a custom message sent to players.
+    cooldownCheck(command, playerName, optionalDisplayCommandName, optionalRawMessage) {
```

Comment:
Wouldn't need both `optionalDisplayCommandName` and `optionalRawMessage`. Because you could use the raw message to put your optional command name.

Not intended to hold up the review

Reply:
thats true i'll go change that